### ISSUE TITLE
Position in module

### ISSUE THREAD

**Issue by Gener4tor** [rubberduck-vba/Rubberduck#4700]
The built-in-Explorer of VBE opens modules at the position they are open:

* DblKlick on Modul "First" => First opens
* scroll down
* DblKlick on Modul "Second" => Second opens
* DblKlick on Modul "First" => First is brought to the top; The position is at the bottom of the modul

This is IMHO the way it should work.

The Code Explorer from rubberduck opens modules always with the cursor on top - whereever it was before. This is very annoying because you have to scroll down and search the position where have been before basicly every time you open a module with the code explorer...

**Comment by Inarion** [comment]
Situational workaround: When having jumped to a declaration with `Shift+F2` you can then jump to your previous position with `Ctrl+Shift+F2`, even across modules. (I'm not totally clear what positions it will jump back to when triggered repeatedly, though.)

That aside, I totally agree. Preserving the cursor positions is something the VBE's project explorer natively does.

**Comment by retailcoder** [comment]
@bclothier that's right - the QMN+Selection for a module (i.e. QualifiedSelection) is always L1C1. To fix it we need to split that instruction and evaluate whether we're looking at a module or a member; if a member, GetNavigationArgs is fine. Otherwise, we need to create a NavigationArgs off the current selection in that module.
